Python/optimizer_analysis.c

```diff
@@ -595,6 +595,13 @@ remove_globals(_PyInterpreterFrame *frame, _PyUOpInstruction *buffer,
         }                              \
     } while (0);
 
+#define ERROR_IF(COND, LABEL) \
+    do {             \
+        if (COND) {  \
+            goto LABEL; \
+        }            \
+    } while (0);
```

Comment:
This is an old pep peeve of mine (see also PEP 8 :-):
```suggestion
#define ERROR_IF(COND, LABEL) \
    do { \
        if (COND) { \
            goto LABEL; \
        } \
    } while (0);
```